Dave2D missing include headers

### LVGL version

v9.2.0

### What happened?

setting "LV_USE_DRAW_DAVE2D" to 1 causes around 450 errors to occur, caused by "incomplete type" errors.

It seems that the necessary private struct type definitions have not been included.
This can be fixed by including the following headers in lv_draw_dave2d.h after "the initial "lv_draw.h" include.
```c
#include "../../lv_draw_private.h"
#include "../../sw/lv_draw_sw.h"
#include "../../lv_draw_label_private.h"
#include "../../lv_image_decoder_private.h"
```

relevant build log snippet:
```
In file included from ../lib/lvgl-9.2.0/src/draw/renesas/dave2d/lv_draw_dave2d_arc.c:1:
../lib/lvgl-9.2.0/src/draw/renesas/dave2d/lv_draw_dave2d.h:33:20: error: field 'base_unit' has incomplete type
   33 |     lv_draw_unit_t base_unit;
      |                    ^~~~~~~~~
../lib/lvgl-9.2.0/src/draw/renesas/dave2d/lv_draw_dave2d.h:53:68: error: unknown type name 'lv_draw_image_dsc_t'
   53 | void lv_draw_dave2d_image(lv_draw_dave2d_unit_t * draw_unit, const lv_draw_image_dsc_t * draw_dsc,
      |                                                                    ^~~~~~~~~~~~~~~~~~~
....
```

### How to reproduce?

You can reproduce this issue by setting up the [ra8d1 lvgl](https://github.com/lvgl/lv_port_renesas_ek-ra8d1_gcc) project and setting the "LV_USE_DRAW_DAVE2D" to 1 in the lv_conf.h file. ( Project is build with [FSP version 5.3.0 in e2studio version 2024-04](https://github.com/renesas/fsp/releases/tag/v5.3.0)

--- a/src/draw/renesas/dave2d/lv_draw_dave2d.h
+++ b/src/draw/renesas/dave2d/lv_draw_dave2d.h
@@ -16,8 +16,17 @@ extern "C" {
 #include "../../../lv_conf_internal.h"
 #if LV_USE_DRAW_DAVE2D
 #include "../../lv_draw.h"
+#include "../../lv_draw_private.h"
 #include "hal_data.h"
 #include "lv_draw_dave2d_utils.h"
+#include "../../lv_draw_rect.h"
+#include "../../lv_draw_line.h"
+#include "../../lv_draw_arc.h"
+#include "../../lv_draw_label.h"
+#include "../../lv_draw_image.h"
+#include "../../lv_draw_triangle.h"
+#include "../../lv_draw_buf.h"
+
 
 /*********************
  *      DEFINES

--- a/src/draw/renesas/dave2d/lv_draw_dave2d_border.c
+++ b/src/draw/renesas/dave2d/lv_draw_dave2d_border.c
@@ -1,6 +1,7 @@
 #include "lv_draw_dave2d.h"
 #if LV_USE_DRAW_DAVE2D
 
+
 static void dave2d_draw_border_complex(lv_draw_dave2d_unit_t * draw_unit, const lv_area_t * outer_area,
                                        const lv_area_t * inner_area,
                                        int32_t rout, int32_t rin, lv_color_t color, lv_opa_t opa);

--- a/src/draw/renesas/dave2d/lv_draw_dave2d_image.c
+++ b/src/draw/renesas/dave2d/lv_draw_dave2d_image.c
@@ -9,6 +9,8 @@
 #include "lv_draw_dave2d.h"
 #if LV_USE_DRAW_DAVE2D
 
+#include "../../lv_image_decoder_private.h"
+
 /*********************
  *      DEFINES
  *********************/

--- a/src/draw/renesas/dave2d/lv_draw_dave2d_label.c
+++ b/src/draw/renesas/dave2d/lv_draw_dave2d_label.c
@@ -1,6 +1,8 @@
 #include "lv_draw_dave2d.h"
 #if LV_USE_DRAW_DAVE2D
 
+#include "../../lv_draw_label_private.h"
+
 static void lv_draw_dave2d_draw_letter_cb(lv_draw_unit_t * draw_unit, lv_draw_glyph_dsc_t * glyph_draw_dsc,
                                           lv_draw_fill_dsc_t * fill_draw_dsc, const lv_area_t * fill_area);
 

